nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractRecordProcessor.java

```diff
@@ -64,6 +64,15 @@
         .required(true)
         .build();
 
+    static final PropertyDescriptor DROP_EMPTY_FILES = new PropertyDescriptor.Builder()
+        .name("drop-empty-files")
+        .displayName("Drop Files With No Records")
```

Comment:
I agree with Joe's point, whether he means it makes sense to have such a property to filter out 0 record files, or that this new property should be consistent in naming and behavior with the one added to QueryRecord. Right now this new property has inverse naming/behavior from the QueryRecord one, should we use a copy of the QueryRecord one here instead?